base.py

```diff
@@ -0,0 +1,124 @@
+import heapq
+import random
+import jsoneng
+from dotenv import load_dotenv
+import os
+
+jdb = jsoneng.JsonDB()
+jdb.create({})
+
+import random
+import heapq
+
+class Character:
+    def __init__(self, name):
+        self.name = name
+        self.state = "idling"
```

Comment:
I prefer the use of enums instead of strings for representing state.

Reply:
we'll do it in the future with the next sprint!